### PR TITLE
Agent to XML Tests

### PR DESCRIPTION
**Pull request by damonkelley**
Partially completes #27

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

File: tests/test_presentation.py [hunk 1]
```diff
@@ -465,4 +471,105 @@ def test_link_object_role(self):
...
+        tree = presentation.objectToAgentXML(agent)
+        agent_xml = etree_to_objectify(tree)
+
+        element = agent_xml.agentType
+        assert element == agent.agent_type
+        assert has_premis_namespace(element)
+
+
+@pytest.mark.django_db
+class TestObjectToPremisAentXML:
```
Comment (somexpert) [inline tests/test_presentation.py]: We should correct the name of this class. Missing the 'g' in Agent

**Comment by somexpert** [comment]
:+1:

**Comment by ldko** [comment]
:+1:

**Pull request merged**